epsagon/modules/tornado.py

```diff
@@ -80,6 +91,10 @@ def after_request(cls, wrapped, instance, args, kwargs):
 
             tornado_runner = cls.RUNNERS.pop(unique_id)
 
+            # Ignoring 404s
+            if getattr(instance, '._status_code', None) == 404:
```

Comment:
Is the dot in the attribute name is correct? shouldn't it be just '_status_code'